### PR TITLE
chore: update content and images

### PR DESCRIPTION
**Pull request by amanji**
*Issue #:* /bcgov/entity#18519

*Description of changes:*

* Updates images and wording to reflex UI designs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DigitalCredentials/CredentialsSimpleSteps.vue [hunk 1]
```diff
@@ -34,7 +34,7 @@
               <div class="align-self-center">2</div>
             </span>
             <span class="ml-8">
-              Scan the QR code with your BC Wallet to get your Digital Business Card and accept it in your BC
+              Scan the QR code with your BC Wallet app to get your Digital Business Card and accept it in your BC
```
Comment (severinbeauvais) [inline src/components/DigitalCredentials/CredentialsSimpleSteps.vue:37]: This reads funny.

Comment (severinbeauvais) [reply src/components/DigitalCredentials/CredentialsSimpleSteps.vue:37]: How about, "In your BC Wallet app, ..."

Comment (amanji) [reply src/components/DigitalCredentials/CredentialsSimpleSteps.vue:37]: Yeah, it's coming from the UI/UX folks.

**Review by severinbeauvais — COMMENTED** [review]
Looks OK to me. Remember to update the app version please.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-586-6hvkk07j.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-586-6hvkk07j.web.app

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**